encord/orm/label_row.py

```diff
@@ -364,3 +366,93 @@ class LabelValidationState(BaseDTO):
     version: int
     is_valid: bool
     errors: List[str]
+
+
+class WorkflowGraphNodeDTO(BaseDTO):
+    uuid: str
+    title: str
+
+
+class LabelRowMetadataDTO(BaseDTO):
+    """
+    Contains helpful information about a label row.
+    """
+
+    label_hash: Optional[str] = Field(default=None, alias="label_uuid")
+    """Only present if the label row is initiated"""
+    created_at: Optional[datetime.datetime] = None
+    """Only present if the label row is initiated"""
+    last_edited_at: Optional[datetime.datetime] = None
+    """Only present if the label row is initiated"""
+    branch_name: str
+    """Only present if the label row is initiated or branch_name is set specifically"""
+
+    data_hash: str = Field(alias="data_uuid")
+    dataset_hash: str = Field(alias="dataset_uuid")
+    dataset_title: str
+    data_title: str
+    data_type: str
+    data_link: Optional[str] = None
+    """Can be `None` for label rows of image groups or DICOM series."""
+    label_status: LabelStatus
+    """Can be `None` for TMS2 projects"""
+    annotation_task_status: Optional[AnnotationTaskStatus] = None
+    """Only available for TMS2 project"""
+    workflow_graph_node: Optional[WorkflowGraphNode] = None
+    is_shadow_data: bool = False
+
+    """Only available for the VIDEO and AUDIO data_type"""
+    frames_per_second: Optional[float]
+    number_of_frames: int
+    duration: Optional[float]
+
+    """Only available for the VIDEO data_type"""
+    height: Optional[int]
+    width: Optional[int]
+
+    """Only available for the AUDIO data_type"""
```

Comment:
Personally I hate having all these optionals that are dependant on an enum and dislike building it in more but not related to this change.